index.js

```diff
@@ -228,6 +281,16 @@ export function load_placements(force_load = false) {
   return Promise.all(
     elements.map((element) => {
       const placement = Placement.from_element(element);
+
+      if (placement && !force_load) {
+        placement.detectABP(ABP_DETECTION_PX, function (usesABP) {
+          placement.uplifted = usesABP;
+          if (usesABP) {
+            console.log("Uses ABP");
```

Comment:
This should probably be more descriptive:

```suggestion
            console.debug("Acceptable Ads enabled. Thanks for allowing our non-tracking ads :)");
```